### PR TITLE
Some update

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #124664
* __->__ #124450
* #124449

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

Differential Revision: [D56440611](https://our.internmc.facebook.com/intern/diff/D56440611)

### CODE REVIEW

File: .ci/docker/ci_commit_pins/executorch.txt [hunk 1]
```diff
@@ -1 +1 @@
-d4b3e5cc607e97afdba79dc90f8ef968142f347c
+554cd2792cf684818b9f33106eb1a6d19bb1f85bq
```
Comment (gmagogsfm) [inline .ci/docker/ci_commit_pins/executorch.txt:1]: ?

File: torch/jit/_trace.py [hunk 2]
```diff
@@ -799,138 +970,26 @@ def forward(self, x):
...
-                _module_class,
-                example_inputs_is_kwarg=isinstance(example_kwarg_inputs, dict),
-            )
-
-    # Allow torch.compile() to inline
-    traced._torchdynamo_inline = func  # type: ignore[attr-defined]
-    return traced
+        traced_mod = WrapperMod(traced_func)
 
+    return torch.export.export(traced_mod, export_example_inputs, example_kwarg_inputs, strict=False).module()
```
Comment (gmagogsfm) [inline torch/jit/_trace.py:992]: how about some correctness verification logic between traced vs exported program?

File: .ci/docker/ci_commit_pins/executorch.txt (hunk 1 above)
Comment (tugsbayasgalan) [reply .ci/docker/ci_commit_pins/executorch.txt:1]: not needed

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D56440611).

**Pull request closed without merging**